SEED-platform/seed
# 1861 rename columns

#### Any background context you want to provide?
If a user maps data into a column (specifically a matching column) and later decides that they don't want the data in that column, there was no way (short of recreating an organization) to remove the data.

#### What's this PR do?
* Add the ability for the user to move data out of the column.
* Note that this may take a while to run with larger datasets (~5 mins for 2000 records). 
* If the user chooses to override an existing column, then the data in that column will permanently disappear.

#### How should this be manually tested?
* Import covered building sample file with default mappings
* view the data (pin relevant columns to the left for ease of viewing changes)
* Go to the org's column settings page and rename a column (e.g., address_line_1) to another name (e.g. new address line 1).
* Go back to the properties list and verify that the address_line_1 data have moved
* Go back to org's column settings page and rename the new column to the previous name.
* Verify that the data return.

#### What are the relevant tickets?
#1861 

#### Screenshots (if appropriate)
